po/ca.po

```diff
@@ -1892,7 +1891,7 @@ msgstr "Mostra la _finestra dels missatges"
 
 #: ../data/geany.glade.h:401
 msgid "Show _Toolbar"
-msgstr "Mos_tra la barra d'eines"
+msgstr "Mostra la barra d’_eines"
```

Comment:
hum actually if changing the "change font" mnemonic required you to change that one, it would be just as fine to change the other one.  And doesn't that conflict with "Mostra els _espais en blanc"?  I know mnemonic can get annoying when you get out of practical letters…
Anyway, don't feel obligated to fix it all just now, but not having duplicates is always better as it allows to activate the item directly instead of only looping through the possible matches.